tools/utils/fiber-helpers.ts

```diff
@@ -1,9 +1,8 @@
-var _ = require("underscore");
-var Fiber = require("fibers");
+import _ from "underscore";
```

Comment:
You don't get a TypeScript error telling you underscore module is not defined here?

Reply:
@trusktr Your `/tools/node_modules/@types` shoud contain a `underscore` folder for type definitions. It was added recently in commit `122f7e4d2d85`.

Reply:
Most of underscore has been removed already. Can you get rid of the rest as well?